tests/unit/server/bundle_manager/mocked_test.py

```diff
@@ -215,18 +229,23 @@ def test_filter_and_sort_workers_tag_exclusive_priority(self):
         sorted_workers_list = self.bundle_manager._filter_and_sort_workers(
             self.workers_list, self.bundle, self.bundle_resources
         )
-        self.assertEqual(len(sorted_workers_list), 2)
+        self.assertEqual(len(sorted_workers_list), 3)
         self.assertEqual(sorted_workers_list[0]['worker_id'], 6)
-        self.assertEqual(sorted_workers_list[1]['worker_id'], 5)
+        self.assertEqual(
```

Comment:
For consistency / clarity, let's just split this up into two different `self.assertEqual` statements.

Reply:
Sorry if I'm missing something obvious, but I'm not sure what you mean, could you clarify?

I added this `assertEqual(set(), set())`because the ordering of these two items is non-deterministic (since ties are broken randomly), so I just wanted to ensure that these two positions hold these two items, but not assert that any item must occur at any particular position.

Reply:
Oh, I didn't realize that the ordering is nondeterministic. Let's just keep it then.